src/sparse.jl

```diff
@@ -9,6 +9,7 @@ end
 size(d::SparseCSCDomain, arg...) = size(d.adomain, arg...)
 length(d::SparseCSCDomain) = Int(d.colptr[end]-1)
 indexranges(d::SparseCSCDomain) = indexranges(d.adomain)
+children(d::SparseCSCDomain) = d.adomain
```

Comment:
I thought I deleted this file.... haha anyway this thing needs a second look. We will need sparsecscdomain back like i mentioned in the last email